driver-puppeteer/index.js

```diff
@@ -9,10 +9,17 @@ const default_url = `file:${__dirname}/index.html`;
 async function mochifyDriver(options = {}) {
   const { stderr = process.stderr, ...launch_options } = options;
 
+  // In case this arrives through CLI flags, yargs will pass a string
+  // when a single arg is given and an Array of strings when multiple
+  // args are given.
+  const extra_args = Array.isArray(launch_options.args)
+    ? launch_options.args
+    : [launch_options.args].filter(Boolean);
```

Comment:
The filter is here so we get `[]` in case `args` is not defined. If this is all too smart, lmk.

Reply:
How about `[].concat(args)`?

Reply:
Oh yeah right, I always forget about the funny animal that is `concat`. Let me change that real quick.

Reply:
Hmm, this would still give us `[undefined]` in case no `args` are given?

Reply:
Ah, yes. So `args ? [].concat(args) : []` maybe?

Reply:
Or how it's done in browserify (where I learned this trick):

```js
[].concat(args).filter(Boolean)
```

Reply:
Done so.